packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -327,7 +327,13 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
               importer,
               index,
               root,
-              normalizeUrl
+              normalizeUrl,
+              async (url, importer) => {
+                const resolveId = await this.resolve(url, importer, {
+                  custom: config.plugins
+                })
+                return resolveId?.id
+              }
```

Comment:
Should we use `config.createResolver` and use the `aliasOnly` option so we resolve with the alias plugin only?